app/main.py

```diff
@@ -1,2 +1,44 @@
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.load_factor = 2 / 3
+        self.size = 0
+        self.hash_table = [[] for _ in range(self.capacity)]
```

Comment:
We actually looking for a different solution for hash table.

> Also, don't forget to store (key, hash, value) as a node in the hash table. A hash table may be stored as a simple list of nodes.

Create class Node as per requirements and make a list of Nodes instead of list of lists. Also It is better to fill table with None values on init. It will be more efficient then creating a thousand empty lists.